plugins/fuzzyhashes/pyproject.toml

```
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "surfactantplugin-fuzzyhashes"
+authors = [
+    {name = "Michael Cutshaw", email = "michael.cutshaw@inl.gov"},
+]
+description = "Surfactant plugin for generating fuzzy hashes"
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ["surfactant"]
+license = {text = "MIT License"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Environment :: Console",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "License :: OSI Approved :: MIT License",
+]
+dependencies = [
+    "python-tlsh",
```

```suggestion
    "py-tlsh",
```

It looks like py-tlsh is the (slightly) more up to date package for TLSH: https://pypi.org/project/py-tlsh/